b2sdk/bucket.py

```diff
@@ -792,6 +801,7 @@ def create_file(
         :param b2sdk.v2.EncryptionSetting encryption: encryption settings (``None`` if unknown)
         :param b2sdk.v2.FileRetentionSetting file_retention: file retention setting
         :param bool legal_hold: legal hold setting
+        :param str,None cache_control: cache control for the new file, optional
```

Comment:
this doesn't really say much. What would one put there, for example? Maybe mention RFC 2616 too.